kratos/spatial_containers/geometrical_objects_bins.h

```diff
@@ -237,6 +237,28 @@ class GeometricalObjectsBins
         return SearchNearestInRadius(ThePoint, max_radius);
     }
 
+    /** This method takes a point and finds the object it is inside.
```

Comment:
this sentence is a bit miss-understandable. Could you try to describe it a bit deeper? Thank you.

Reply:
I have modified the sentence, thanks for you feedback. Tell me if it's clear enough.